core/src/main/java/io/grpc/LoadBalancer2.java

```diff
@@ -317,6 +322,77 @@ public String toString() {
   }
 
   /**
+   * Provides arguments for a {@link SubchannelPicker#pickSubchannel(PickSubchannelArgs)}.
+   */
+  public abstract static class PickSubchannelArgs {
```

Comment:
I'm confused. The final immutable class still makes sense to me in this context, no? Do you anticipate passing different types of `PickSubchannelArgs` to `pickSubchannel` internally? If so when and why?